### ISSUE TITLE
PNI - Share Button Mockup

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/foundation.mozilla.org#5716]
**Background:**
One of the PNI 2020's goal is: 10% more mention/shares on social media
In this 202 iteration, we removed the “copy link and share” and @sabrina raised the concern that we were removing the only visible share mechanism, which could work against the goal we set.

We did further research (#5494) to explore this topic and learned that:
- People used the button “copy link and share” (2019 version) more frequently than the “share” button hidden under “vote” (in 2019, the conversion rate for “Copy Link Tap” was 0.19% vs the “Share” under vote, which was 0.11%). 
-   We think that the easier we make for people to share (and set some “reminders” through the site), the more likely it is they will share. **Note:** perhaps it can be done with copy as to not pollute/distract the interface with too much either. 
-   All of the benchmarks researched showed visible share buttons on the interface.
-   According to our Google Analytics, Facebook and Twitter are popular social media sites bringing traffic to our PNI pages. We also see those 2 social media buttons in our benchmark. That makes them the ideal candidates to be present in our site. 
-   We may be careful to not pollute the interface and deviate from the main goal that is to inform people. For this reason the recommendation is to add the share button either in the middle or bottom of product pages. From the main page, we could ask people to share the guide at the end of the page - the challenge here is to find the right opportunity for this type of ask.

**To do:**
- [ ] Reach out to Audrey to hear her thoughts on what types of social media buttons would be useful for PNI audience. From the research, we see LinkedIn emerging as a potential Social media platform for us in 2020. Also we could bring insta more if relevant. It would also be good to connect with her to see if she has any insights related to Social Media performance and how those match with traffic from social we see in GA. 
- [ ] Do mockups with the share buttons (home and product pages) and get approval by stakeholders if needed.
- [ ] Implement share buttons + add trackers ([spreadsheet here](https://docs.google.com/spreadsheets/d/1x-CP_HYaAs3oRv2cjbSFu8w5Z938E4cyHQ3hvh17c6c/edit?usp=sharing)) - OPEN TICKET
- [ ] Measure on GA if they are making a difference in shares + circle back with Audrey to see if there are any improvements - OPEN TICKET

cc: @jamilasnell @kristinashu tagging you both for triage.

**Comment by Pomax** [comment]
@tais I've assigned this to you to figure out who should be working on this next, so we don't lose track of this issue.

**Comment by taisdesouzalessa** [comment]
@kristinashu do we have any new PNI hub coming up so we can try out the changes here? Or maybe we should do them to the current site? Not sure where we can fit this work.

**Comment by kristinashu** [comment]
This doesn't need to be blocked by a new PNI edition release, it could be done whenever we have time.

**Comment by Pomax** [comment]
@kristinashu do we want to keep this issue open, or do we want to treat this as new work with a new issue whenever there is a new design round for PNI?
